flut/system_tools.cpp

```diff
@@ -71,13 +71,25 @@ namespace flut
 		return ifs.good();
 	}
 
+
 	FLUT_API string get_date_time_str( const char* format )
 	{
-		auto in_time_t = std::chrono::system_clock::to_time_t( std::chrono::system_clock::now() );
+        // GCC did not implement std::put_time until GCC 5
+#if defined(__GNUC__) && (__GNUC__ < 5)
+        auto in_time_t = std::chrono::system_clock::to_time_t( std::chrono::system_clock::now() );
+        char arr[100];
+
+        std::stringstream ss;
+        ss << std::strftime(arr, sizeof(arr), format, std::localtime(&in_time_t));
+        return ss.str();
+#else
```

Comment:
Doesn't the result get stored in `arr`? If so, you can directly create a string from that and omit the `stringstream`.